packages/server/src/app.module.ts

```diff
@@ -19,12 +18,10 @@ import { join } from 'path';
       driver: ApolloDriver,
       autoSchemaFile: join(process.cwd(), 'dist/schema.gql')
     })
-  ],
-  controllers: [AppController],
-  providers: []
+  ]
 })
 export class AppModule implements NestModule {
   configure(consumer: MiddlewareConsumer): void {
-    consumer.apply(ProjectGuard).exclude('/(health|projects|graphql)(/.*)?').forRoutes('/');
+    consumer.apply(ProjectGuard).exclude('/', '/(health|projects|graphql)(/.*)?').forRoutes('/');
```

Comment:
Will this not just exclude everything?

Reply:
Just going to revert this, cause I was trying todo static side render, but went with nginx